morebits.js

```diff
@@ -1249,14 +1249,28 @@ Morebits.unbinder.getCallback = function UnbinderGetCallback(self) {
  * is fairly unlikely that anyone will iterate over a Date object.
  */
 
+Date.monthNames = ['January', 'February', 'March', 'April', 'May', 'June',
+	'July', 'August', 'September', 'October', 'November', 'December' ];
+
+Date.monthNamesAbbrev = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'];
+
+Date.prototype.getMonthName = function() {
```

Comment:
I suppose these could remain absent for now. Unless we need them back?

Reply:
Per #635, yeah.  A global search shows that only kowiki actually uses it (also ro but IIRC ko works) but otherwise unused.